lua/lspconfig/jsonls.lua

```diff
@@ -11,7 +11,7 @@ configs[server_name] = {
     init_options = {
       provideFormatter = true,
     },
-    root_dir = util.root_pattern(".git", vim.fn.getcwd()),
+    root_dir = util.find_git_ancestor,
```

Comment:
Having `cwd` as a fallback is great here. The `JSON` LSP doesn't even have to deal with things like imports or alike, it _just_ checks the current file.

Reply:
get_cwd() is completely incorrect, it should be, at best, util.path.dirname